admin/spam/views.py

```diff
@@ -1,9 +1,42 @@
-from django.http import HttpResponse
+from django.shortcuts import render, render_to_response
+from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
+from django.contrib.auth.decorators import login_required
 
+from .serializers import serialize_comments, retrieve_comment
 
+
+@login_required
 def spam_list(request):
-    return HttpResponse('This is a list of spam')
+    comment_list = serialize_comments()
```

Comment:
@sloria I switched this to resemble more of the prereg with only serializing the current items on the page. Would you like this to go further and actually implement `limit` and `offset` on the database? The pagination on the Django side will not be able to use the available solutions for Django pagination.